client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -146,7 +148,7 @@ class CalypsoifyIframe extends Component< Props & ConnectedProps & ProtectedForm
 			this.iframeRef.current.contentWindow
 		) {
 			this.successfulIframeLoad = true;
-			const { port1: iframePortObject, port2: transferredPortObject } = new MessageChannel();
+			const { port1: iframePortObject, port2: transferredPortObject } = new window.MessageChannel();
```

Comment:
Nit - Wondering what happened to the linter here 🤔 you don't need to grab `MessageChannel` off the window object - better to use the global for constructing an instance, and the `window` object property to verify support. 

Might be better to define a global exception for eslint instead of changing this line, considering how rarely MessageChannel is used.

Reply:
@jeraldjuice  See #37953 for more details about what happened to the linter 😉 Using "naked" globals is now discouraged as there's too many of them on `window` (700+) and their usage can hide serious bugs when referencing an undefined variable.

Reply:
@jsnajdr gotcha, that clarifies this change 👍🏻